docs/_docs/refs/mojito-locales.md

```diff
@@ -115,12 +119,15 @@ permalink: /docs/refs/mojito-locales/
 | ms-BN	        | Malay (Brunei Darussalam) |
 | ms-MY	        | Malay (Malaysia) |
 | mt-MT	        | Maltese (Malta) |
+| mus-021	        | Creek (Northern America) |
```

Comment:
Muscogee (Creek). Creek is the historical name, Muscogee (Creek) is the nation

Reply:
I think Muscogee works fine here

Reply:
Honestly, this is the small tedious stuff that I'd be happy to wade through the bureacracy on -- "Creek" is what's in CLDR dataset that's maintained internationally, so it'll keep showing up if it's wrong, since much of the field of localization seems programmatic or at least highly inclined to follow the standards:
https://github.com/unicode-cldr/cldr-localenames-full/blob/993632df2f5d6a2d33cbbf40d922474c2482eaca/main/en-001/languages.json#L390

If you can confirm this is universal that "Muscogee" is the more appropriate term, then I could wrangle with the bureacracy to have that reflected in the standards. We could review the standards and collect a list of stuff like this to push through en masse

Reply:
@arecvlohe this is great info, thanks for sharing

Reply:
Oh hey, good news: seems someone else chimed in on this in Aug 2009, and it's in the upcoming release: https://unicode-org.atlassian.net/browse/CLDR-13193?jql=text%20~%20%22muskogee%22

Reply:
Oh nice! I guess I don't have to do anything then.